src/ansys/sound/core/sound_composer/_source_control_parent.py

```diff
@@ -21,9 +21,18 @@
 # SOFTWARE.
 
 """Sound composer's source control."""
+from enum import Enum
+
 from ._sound_composer_parent import SoundComposerParent
 
 
+class SpectrumSynthesisMethods(Enum):
+    """Class providing a list of available spectrum synthesis methods."""
+
+    IFFT = 1
```

Comment:
@anssmolla or @mattbennett72 could you please suggest some documentation for these two methods here ? One or two lines per method should be enough and I believe it has to be similar to what can be found in https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/corp/v241/en/Sound_SAS_UG/Sound/UG_SAS/methods_for_sound_creation_184484.html

(@ansaminard FYI)

Reply:
@a-bouth I added some docstrings in the meantime. @anssmolla I'll let you check it